app/renderer/lib/client-frameworks/robot.js

```diff
@@ -0,0 +1,102 @@
+import Framework from './framework';
+
+class RobotFramework extends Framework {
+
+  get language () {
+    //TODO: Make https://highlightjs.org/ use robot syntax
+    return "python";
+  }
+
+  get getCapsVariables() {
+    return Object.keys(this.caps).map((k) => {
+      return `${k}    ${this.getPythonVal(this.caps[k])}`;
+    }).join("\n");
+  }
+
+  getPythonVal (jsonVal) {
+    if (typeof jsonVal === 'boolean') {
+      return jsonVal ? "True" : "False";
+    }
+    return jsonVal;
+  }
+
+  wrapWithBoilerplate (code) {
+    return `# This sample code uses the Appium robot client
+# pip install robotframework-appiumlibrary
```

Comment:
is robot framework only in python? or should this be called "RobotFramework Python"?